src/Components/styles.css

```diff
@@ -0,0 +1,336 @@
+@font-face {
```

Comment:
you seem to have the old version of `styles.css` which has all the css code in a single file. The current version of this css file has been refactored, so that we can have separate, smaller files for each component. Can you put your css code in a separate file named after your component?

Reply:
I just removed the styles.css file from my branch. Right now, the component itself doesn't have any css styling, but I made a Blob.css that can be used for positioning blobs on each page.